modules/navier_stokes/test/tests/finite_volume/cns/stagnation_inlet/supersonic_nozzle_hllc.i

```diff
@@ -0,0 +1,315 @@
+stagnation_pressure = 1
+stagnation_temperature = 1
+
+[GlobalParams]
+  fp = fp
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
+
+[Debug]
+   show_material_props = true
+[]
+
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    file = supersonic_nozzle.e
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [fp]
+      type = IdealGasFluidProperties
+      allow_imperfect_jacobians = true
```

Comment:
is that a thing for AD?

Reply:
There's this in `FluidProperites::validParams`:
```
  params.addParam<bool>(
      "allow_imperfect_jacobians",
      false,
      "true to allow unimplemented property derivative terms to be set to zero for the AD API");
```
However, this doesn't actually appear to be necessary for this input (nor should it ever be for FV (which is always AD)) so I will remove this